plugins/apps/threatbus_misp/Makefile

```diff
@@ -0,0 +1,37 @@
+colon := :
+$(colon) := :
+
+.PHONY: all
+all: format build dist test
+
+.PHONY: format
+format:
+	python -m black .
+
+.PHONY: test
+test: unit-tests
+
+.PHONY: unit-tests
+unit-tests:
+	python -m unittest discover .
+
+.PHONY: clean
+clean:
+	find . -type d -name __pycache__ -exec $(RM) -r {} \;
```

Comment:
Not quite sure where `RM` comes from, but might not be important for this review.

Reply:
`$(RM)` is a Makefile [variable](https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html).